README.md

```diff
@@ -37,6 +42,38 @@ Instead of specifying the packages directly, you can pass a file containing the
            package_file: tl_packages
 ```
 
+### Enabling updating cache using a workflow trigger
+
+One can inclue a cache referesh using GitHub's `workflow_dispatch` functionality:
+
+```yaml
+name: check
+on:
+  push:
+  pull_request:
```

Comment:
Running this on `push` and `pull_request` seems weird since it wouldn't have the `cache_version` parameter there and the regular pipeline should refresh the cache anyway.

Reply:
In that case, the empty string is passed as parameter, which "magically" works (then they key at that "area" is then `__` instead of `_0_`).